### ISSUE TITLE
Cannot authenticate myself using interactive login.

### ISSUE THREAD

**Issue by HouzuoGuo** [superfly/flyctl#676]
**Describe the bug**

Hello! I'm using `flyctl auth login -i` to authenticate via interactive username and password input. I do not use one time password.

The password input should be correct - it works on fly.io website. I've made at least 15 login attempts so the likelihood of an incorrect password input is small. All attempts fail with the following output:

```
> ~/.fly/bin/flyctl auth login -i
WARN no config file found at /home/xxxx/config.yml
? Email: guohouzuo@gmail.com
? Password: *********************
? One Time Password (if any):
Error failed retrieving access token: Incorrect email and password combination
```

The flyctl version reads:

```
flyctl v0.0.270 linux/amd64 Commit: 7da8b26 BuildDate: 2021-12-17T22:27:41Z
```

** Add `fly.toml`

This is a fresh installation of flyctl. The file does not yet exist.

**Comment by jsierles** [comment]
Thanks for the report. Are you able to login without `-i`?

**Comment by HouzuoGuo** [comment]
Happy new year!

The flyctl CLI command runs on a headless server, so it won't be able to perform a browser-based login on that particular server host. However I've managed to perform a browser-based login on a Linux desktop and then copy the `config.yaml` file through to the headless server.

**Comment by azazeal** [comment]
Apologies, this is a verified bug which has been patched in `v0.0.272-pre-3`. I expect that a release which contains the fix will be out in the near future.

**Comment by HouzuoGuo** [comment]
Thanks!
